src/webextension/list/popup/components/app.css

```diff
@@ -13,6 +13,6 @@ body {
 }
 
 .app {
-  width: 250px;
+  width: 320px;
```

Comment:
This is probably something for @changecourse to think on, but I'm of two minds about this:

1. It's good that this is consistent with the Unlock doorhanger
2. This might be too wide for a usable interface; if the doorhanger is so wide that it covers up the login form on a page, it reduces the usability of the doorhanger because now you have do the following:
    1. Open the doorhanger
    2. Copy the password (at least 1 click when we have URL prefilling, but at least 2 right now)
    3. Close the doorhanger
    4. Right-click on the field
    5. Select Paste
   ---
    With a narrower doorhanger, you can merge (iii) and (iv) together, reducing the number of clicks by 1. On a 720p monitor, 320px is *just enough* to force users to take the longer path to pasting their password into Facebook. Here's an example of the issue. You have about 1 or 2 pixels where you can actually click on the password field while the doorhanger is open; if we reduced this by even 20 pixels, it would be a significant usability improvement for a very popular website.

    ![too-wide](https://user-images.githubusercontent.com/826865/37055369-6cd5c750-2136-11e8-824d-409afdd0938f.png)